swarm/api/http/server.go

```diff
@@ -454,57 +464,32 @@ func (s *Server) HandlePostResource(w http.ResponseWriter, r *Request) {
 				Respond(w, r, fmt.Sprintf("cannot resolve %s: %s", r.uri.Addr, err), http.StatusNotFound)
 				return
 			}
-		} else {
-			w.Header().Set("Cache-Control", "max-age=2147483648")
 		}
 
 		// get the root chunk key from the manifest
-		addr, err = s.api.ResolveResourceManifest(manifestAddr)
+		rootAddr, err = s.api.ResolveResourceManifest(manifestAddr)
```

Comment:
"Cache-Control" header setting ommited on purpose?

Reply:
What do you mean? `Cache-Control` does not make sense in POST requests.